helm_deploy/apply-for-legal-aid/requirements.yaml

```diff
@@ -1,5 +1,5 @@
 dependencies:
 - name: postgresql
-  version: ^0.8.3
+  version: ^6.2.0
   condition: postgresql.enabled
-  repository: https://kubernetes-charts.storage.googleapis.com/
+  repository: https://charts.bitnami.com
```

Comment:
The original PostgreSQL image runs as root so we are now using bitnami's PostgreSQL
as recommended by cloud-platform.
It's also what is used by `check-financial-eligibility` at the moment